client/src/components/navbar/navbar.jsx

```diff
@@ -76,8 +97,18 @@ export default function Navbar() {
       >
         <Toolbar className={classes.toolbar}>
           <Typography variant="h6" noWrap className={classes.toolbarTitle}>
-            <Link href="/" className={classes.link}>
-              Baking Company
+            <Link href="/" className={classes.logolink} style={{}}>
+              <img
+                src={
+                  process.env.PUBLIC_URL +
```

Comment:
Could use a template string here to improve readability.
`${process.env.PUBLIC_URL}/Assets/images/birthday-cake-solid.svg`

Reply:
I believe we can actually import image files directly in Javascript. That is recommended as it'll optimize the importing. 

We then also don't need the PUBLIC_URL here. And either way, you should probably be using a relative as opposed to an absolute path for these.

Reply:
See this for reference - https://create-react-app.dev/docs/adding-images-fonts-and-files
You can do this ^ or do relative paths. I don't want to see process.env.PUBLIC_URL in React

Reply:
But, how you did it in the index file is right, that is based on how webpack compiles create-react-app